backend/utils/rabbitmq_client.py

```diff
@@ -0,0 +1,64 @@
+import os
+import aio_pika
+
+RABBITMQ_USER = os.getenv('RABBITMQ_DEFAULT_USER')
+RABBITMQ_PASS = os.getenv('RABBITMQ_DEFAULT_PASS')
+RABBITMQ_HOST = str(os.getenv('RABBITMQ_HOST'))
+RABBITMQ_PORT = int(os.getenv('RABBITMQ_PORT'))
+RABBITMQ_QUEUE_USER_CHATS = os.getenv('RABBITMQ_QUEUE_USER_CHATS')
+RABBITMQ_QUEUE_USER_CHAT_REPLIES = os.getenv('RABBITMQ_QUEUE_USER_CHAT_REPLIES')
+RABBITMQ_QUEUE_MAGIC_LINKS = os.getenv('RABBITMQ_QUEUE_MAGIC_LINKS')
```

Comment:
We do not need a separate queue for magic links. The backend is responsible to generate a chat message and as far as the channel bots is concerned it is just another chat message. Otherwise we back knowledge of the login process into the channel bots and there is no benefit to that.